public/app/features/alerting/unified/components/rule-editor/RuleInspector.tsx

```diff
@@ -111,6 +127,25 @@ const InspectorYamlTab: FC<YamlTabProps> = ({ onSubmit }) => {
   );
 };
 
+function YamlContentInfo() {
+  return (
+    <div>
+      The YAML content in the editor represents a single alert only. <br />
+      It does not contain the complete configuration of alert rules needed for Prometheus configuration
```

Comment:
@brendamuir Could you take a look at this message? We'd like to be clear that we display only a part of the necessary YAML configuration

Reply:
A single alert / alert rule? Do you wnat to state that in the first part of this sentence - like "The YAML content in the editor only displays part of the configuration required for an alert rule.(Prometheus as well - or something else?) It does not contain the complete configuration required for the Prometheus configuration of an alert rule. " does that make sense?

Reply:
Some customers may use this YAML editor to copy the content and then use it as the content of a Prometheus configuration file. 
What I want to emphasize is the content contains only an alert rule and if you want to use that as a config file, then you should be aware that this is only part of the configuration file

Reply:
The YAML content in the editor only contains alert rule configuration. To configure Prometheus, copy the entire configuration file.